playwright/tests/404page.spec.js

```diff
@@ -1,8 +1,14 @@
-const { test, expect } = require('@playwright/test')
+const { test, expect } = require('../utils/next-test')
 
 test.describe('404 Error Page', () => {
+  test.beforeEach(async ({ context }) => {
```

Comment:
This chunk is repeating in each of the following tests. Can it be DRY-ed up?

Reply:
all DRYed up!